server/game/cards/19.1-AS01/Dragon/SeijisFate.js

```diff
@@ -7,7 +7,7 @@ class SeijisFate extends DrawCard {
             effect: [
                 AbilityDsl.effects.blank(),
                 AbilityDsl.effects.addTrait('creature'),
-                AbilityDsl.effects.loseTrait('courtier')
+                AbilityDsl.effects.loseTraits(['bushi', 'courtier'])
```

Comment:
This should probably be two different `loseTrait` effects rather than combining them like this.